packages/playground/website/src/main.tsx

```diff
@@ -131,6 +131,53 @@ function Modals() {
 	return null;
 }
 
+/**
+ * Allow the parent window to execute PHP code in Playground.
+ *
+ * ## Example:
+ *
+ * ### Send a message to the Playground iframe:
+ * ```
+ * const frame = document.getElementById('iframe-id');
+ * frame.contentWindow.postMessage({ type: 'php-request', code: `<?php echo 'Hey Playground!';` }, '*');
+ * ```
+ *
+ * ### Receive a message from the Playground iframe:
+ * ```
+ * window.addEventListener('message', (event) => {
+ * 	if(event.data.type === 'php-response') {
+ *		console.log('Message received', event.data.response);
+ * 	}
+ * });
+ *
+ */
+function addPhpRequestListener(playground: PlaygroundClient) {
+	window.addEventListener('message', async (event) => {
+		if (event.data.type === 'php-request') {
```

Comment:
Could we model this after the Blueprint API and call it `runPHP`, and ultimately, allow any/most blueprint steps?

Reply:
I would model it based on the [Playground API Client](https://wordpress.github.io/wordpress-playground/javascript-api/playground-api-client/), because we started with the goal of allowing people do JS API interactions when the API isn't available. 

If some features are missing, we could expand both.

Reply:
Yes, that's where we started. But thinking from my usecase, it would be very useful to be able to simply install a plugin. Is it feasible to add a command to execute a blueprint step or a big deal?

Reply:
```
runBlueprintSteps(
	compileBlueprint(event.data.blueprint),
	playground
);
```

This code would run a blueprint.

Reply:
The problem is that it wouldn't return an output.

Reply:
Likely that's ok. For just most activities like creating a post, I don't really need to check the return code anyway.